lib/src/Base/Graph/Polygon.cxx

```diff
@@ -169,4 +169,29 @@ void Polygon::load(Advocate & adv)
   adv.loadAttribute( "edgeColor_", edgeColor_ );
 }
 
+/* Builds a polygon which fills the area between two curves */
+Polygon Polygon::FillBetween(Sample const& dataX, Sample const& dataY1, Sample const& dataY2)
```

Comment:
You don't want pass const references to samples (meaning that the memory address will not change), but rather pass references to const samples (meaning that the sample will not be changed and avoiding the copy of a large object using references). So you should use:
```
Polygon Polygon::FillBetween(const Sample & dataX, const Sample & dataY1, const Sample & dataY2)
```
The reference operator & is with a blank on both side in OT.

Reply:
Even though the meaning is not changed, I have replaced the line with yours.